projects/storefrontlib/src/shared/directives/auto-focus/auto-focus.directive.ts

```diff
@@ -0,0 +1,12 @@
+import { AfterViewInit, Directive, ElementRef } from '@angular/core';
+
+@Directive({
+  selector: '[cxAutoFocus]',
+})
+export class AutoFocusDirective implements AfterViewInit {
```

Comment:
which one is better here? AfterViewInit or AfterViewChecked?

Reply:
So in conclusion can we leave it as is?

Reply:
Yup